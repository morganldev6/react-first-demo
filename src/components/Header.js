import React from "react";

const Header = ({ title }) => {
  return (
    <header>
      <div className="wrapper">
        <h1>Star Wars Movies</h1>

        <p>
          This site demonstrates the use of a WordPress headless CMS for backend
          with a React frontend built on top.
        </p>
      </div>
    </header>
  );
};

export default Header;
