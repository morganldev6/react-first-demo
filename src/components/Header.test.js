import React from "react";
import { render, cleanup } from "@testing-library/react";

import Header from "./Header";

afterEach(cleanup);

describe("Footer component", () => {
  test("Renders correctly", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
