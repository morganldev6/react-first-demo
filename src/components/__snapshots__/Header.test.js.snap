// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders correctly 1`] = `
<div>
  <header>
    <div
      class="wrapper"
    >
      <h1>
        Star Wars Movies
      </h1>
      <p>
        This site demonstrates the use of a WordPress headless CMS for backend with a React frontend built on top.
      </p>
    </div>
  </header>
</div>
`;
